airbyte-integrations/connectors/source-linkedin-leads/source_linkedin_leads/manifest.yaml

```diff
@@ -0,0 +1,183 @@
+version: 2.0.0
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - lead_form
+    - lead_form_response
+
+definitions:
+  schema_loader:
+    type: JsonFileSchemaLoader
+    file_path: "./source_linkedin_leads/schemas/{{ parameters['name'] }}.json"
+
+  streams:
+    lead_form:
+      type: DeclarativeStream
+      name: lead_form
+      retriever:
+        type: SimpleRetriever
+        paginator:
+          type: DefaultPaginator
+          page_size_option:
+            type: RequestOption
+            field_name: count
+            inject_into: request_parameter
+          page_token_option:
+            type: RequestOption
+            field_name: start
+            inject_into: request_parameter
+          pagination_strategy:
+            type: OffsetIncrement
+            page_size: 1000
+            inject_on_first_request: true
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /leadForms?owner=(sponsoredAccount:urn%3Ali%3AsponsoredAccount%3A{{stream_partition.account_id }})
+          http_method: GET
+          request_headers:
+            LinkedIn-Version: "202406"
+            X-Restli-Protocol-Version: 2.0.0
+          request_parameters:
+            q: owner
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - elements
+        partition_router:
```

Comment:
Is that the same account_id as in linkedin_ads?

Reply:
@natikgadzhi yes, why ?